content/en/database_monitoring/setup_oracle/exadata.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Setting Up Database Monitoring for Oracle RAC
+kind: documentation
+description: Install and configure Database Monitoring for Oracle RAC
+private: true
+is_beta: true
+further_reading:
+- link: "/integrations/oracle/"
+  tag: "Documentation"
+  text: "Basic Oracle Integration"
+
+---
+
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">Database Monitoring is not supported for this site.</div>
+{{< /site-region >}}
+
+<div class="alert alert-info">
+The features described on this page are in private beta. Contact your Customer Success Manager for information on the required Agent build and installation instructions.
+</div>
+
+Database Monitoring provides deep visibility into your Oracle databases by exposing query samples to profile your different workloads and diagnose issues.
+
+## Before you begin
+
+Supported Oracle versions
+: 19c, 21c
+
+Supported Agent versions
+: For more information on the required Agent build, contact your Customer Success Manager.
```

Comment:
Let's add the Agent build / installation information here so that we don't have to be going back and forth with the CSMs

Reply:
I added the latest Agent version in the main (index) page. Also, I removed the supported agent & db versions from each setup, because the information is already contained in the index page.

Reply:
Removed it.